app/controllers/api/metrics_controller.rb

```diff
@@ -68,11 +52,7 @@ def destroy
       flash[:error] = 'The Hits metric cannot be deleted'
     end
 
-    respond_to do |format|
-      format.html do
-        redirect_to admin_service_metrics_path(@service, tab: "#{method_or_metric}s")
-      end
-    end
+    redirect_to admin_service_metrics_path(@service, tab: "#{method_or_metric}s")
```

Comment:
Why do we get rid of `respond_to`? We do the same regardless of format requested?

Reply:
Rails responds with HTML by default and we don't require any other format for these actions, so I figured we could reduce the verbosity.

Reply:
if another format is requested, will the error message change after this update?

Reply:
well, this is a potential issue. What is the old and new message?

Reply:
Sorry I meant the opposite, the response is exactly the same.